README.MD

```diff
@@ -0,0 +1,114 @@
+### @password-validator/core
+
+<div style="display:flex; gap:10px; flex-wrap:wrap">
+  <a href="https://www.npmjs.com/package/@password-validator/core" style="text-decoration:none">
+    <img src="https://img.shields.io/npm/v/@password-validator/core.svg"  />
+  </a>
+
+  <a href="https://www.npmjs.com/package/@password-validator/core" style="text-decoration:none">
+    <img src="https://img.shields.io/npm/dt/@password-validator/core.svg" />
+  </a>
+
+  <a href="https://github.com/Theshedman/password-validator.js" style="text-decoration:none">
+    <img src="https://img.shields.io/github/workflow/status/Theshedman/password-validator/build-ci?logo=github" />
+  </a>
+
+  <a href="https://github.com/Theshedman/password-validator.js" style="text-decoration:none">
+    <img src="https://img.shields.io/github/languages/top/Theshedman/password-validator.js.svg"  />
+  </a>
+</div>
```

Comment:
You can as well add the code coverage result here too. This will give the users of the package more confidence that the package is well tested and good to use.

Reply:
Couldn't actually find a link to that. Will dig deeper

Reply:
It's okay for now.